<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1002</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <property name="horizontalSpacing">
       <number>14</number>
      </property>
      <property name="verticalSpacing">
       <number>50</number>
      </property>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>50</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>50</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionScreen1"/>
   <addaction name="separator"/>
   <addaction name="actionScreen2"/>
   <addaction name="separator"/>
   <addaction name="actionScreen3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionScreen1">
   <property name="text">
    <string>Screen1</string>
   </property>
  </action>
  <action name="actionScreen2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Screen2</string>
   </property>
  </action>
  <action name="actionScreen3">
   <property name="text">
    <string>Screen3</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
